routes/temperature_anomalies.py

```diff
@@ -0,0 +1,171 @@
+import asyncio
+from math import floor
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+)
+
+# local imports
+from fetch_data import (
+    describe_via_wcps,
+)
+from csv_functions import create_csv
+from validate_request import (
+    get_coverage_encodings,
+    construct_latlon_bbox_from_coverage_bounds,
+    validate_latlon_in_bboxes,
+)
+from generate_urls import generate_wcs_query_url
+from generate_requests import generate_wcs_getcov_str
+from fetch_data import fetch_data, describe_via_wcps
+from postprocessing import merge_dicts, postprocess, prune_nulls_with_max_intensity
+from . import routes
+
+seaice_api = Blueprint("temperature_anomalies_api", __name__)
```

Comment:
Guessing this route was seeded with the sea ice route!

```suggestion
temperature_anomaly_api = Blueprint("temperature_anomalies_api", __name__)
```

or, since this doesn't really get called anywhere we could do something like 

```suggestion
_temperature_anomaly_api = Blueprint("temperature_anomalies_api", __name__)
```